src/collections/blog/2022/2022-20-06-managing-multiple-clusters-with-meshery/index.mdx

```diff
@@ -0,0 +1,99 @@
+
+## Using multi cluster with Meshery
+From a client's perspective, there are two uses of the multi context feature in general. While deploying a <Link to="/meshmap">MeshMap</Link> design or performing any other operation on their cluster(s), selecting any number of kubernetes contexts will allow them to uniformly and parallely perform the operation across the clusters. And while visualizing the state of their cluster(s), the same context switcher will allow them to filter across the clusters whose view they want to see.
+
+
+All cluster specific operations are now applied over a number of clusters uniformly. So if you have 10 clusters to manage and 8 of those start with the exact same set of pods, deployments, service mesh, etc then <Link to="/meshery">Meshery</Link> can help you to apply these operations quickly and easily.
+
+
+It is as simple as selecting the specific cluster(s) from the kubernetes context switcher in the navbar, and then applying whatever operation you wanted to, whether that be deploying a sample app, a service mesh, or a <Link to="/meshmap">MeshMap</Link> design.
+
+<p align="center">
+  <img src={Switcher} alt="context switcher" class="image-center-shadow" />
+</p>
+
+Just before applying the operation, users will be prompted with a confirmation modal which will provide the information about which cluster(s) that operation will be performed against. As the User interface improves, this same modal will also convey more useful information about the operation they are going to perform.
+
+<p align="center">
+  <img src={Deploy} alt="deploy modal"  class="image-center-shadow" />
+</p>
+
+
+<br/>
+
+### Using Meshmap visualizer
+You can switch between views of your cluster in visualizer mode while using <Link to="/meshmap">MeshMap</Link>.
+
+<p align="center">
+  <img src={Cluster2} alt="visualizer showing data of context1" class="slides-right" />
+</p>
+<p align="center">
+  <img src={Cluster1} alt="visualizer showing data of context2" class="slides-left" />
```

Comment:
A poor second `alt` description (and first, come to think of it). Please be more descriptive in both `alt`s.